src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantsController.kt

```diff
@@ -0,0 +1,170 @@
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.ApiResponse409
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.customer.db.OrganizationStore
+import com.terraformation.backend.customer.db.ProjectStore
+import com.terraformation.backend.db.accelerator.CohortId
+import com.terraformation.backend.db.accelerator.ParticipantId
+import com.terraformation.backend.db.default_schema.OrganizationId
+import com.terraformation.backend.db.default_schema.ProjectId
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.ArraySchema
+import io.swagger.v3.oas.annotations.media.Schema
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/participants")
+@RestController
+class ParticipantsController(
+    private val cohortStore: CohortStore,
+    private val organizationStore: OrganizationStore,
+    private val participantService: ParticipantService,
+    private val participantStore: ParticipantStore,
+    private val projectStore: ProjectStore,
+) {
+  @ApiResponse200
+  @Operation(summary = "Creates a new participant.")
+  @PostMapping
+  fun createParticipant(
+      @RequestBody payload: CreateParticipantRequestPayload
+  ): GetParticipantResponsePayload {
+    val model =
+        participantService.create(
+            ParticipantModel.create(cohortId = payload.cohortId, name = payload.name))
+
+    return makeGetResponse(model)
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @ApiResponse409("There are projects associated with the participant.")
+  @DeleteMapping("/{participantId}")
+  @Operation(summary = "Deletes a participant that has no projects.")
+  fun deleteParticipant(@PathVariable participantId: ParticipantId): SimpleSuccessResponsePayload {
+    participantStore.delete(participantId)
+
+    return SimpleSuccessResponsePayload()
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{participantId}")
+  @Operation(summary = "Gets information about a participant and its assigned projects.")
+  fun getParticipant(@PathVariable participantId: ParticipantId): GetParticipantResponsePayload {
+    val model = participantStore.fetchOneById(participantId)
+
+    return makeGetResponse(model)
+  }
+
```

Comment:
will we also want a GET / for retrieving a list of participants? Or maybe we can use search for that.

Reply:
We can add one if we need it, but the only places I saw where we would need to get lists of participants were either table views with CSV export (which means they need to use search anyway) or participant selection that's covered by `GET /api/v1/accelerator/organizations`.